README.md

```diff
@@ -1,16 +1,17 @@
 #monasca-agent
 Installs the [monasca-agent](https://github.com/stackforge/monasca-agent) part of the [Monasca](https://wiki.openstack.org/wiki/Monasca) project.
+It is installs it into a virtualenv on the box.
 
-##Requirements
+## Requirements
 - keystone_url:
-- monasca_agent:
-  user:
-  password:
-  project:
-  service:  # Defining the service is optional
-  dimensions: 'role:monitoring,region:a' # Defining dimensions is optional
+- monasca_agent_user:
+- monasc_agent_password:
```

Comment:
You have monasc_ instead of monasca_ for these two variables and the first two under Optional